test/helpers.bash

```diff
@@ -101,11 +101,7 @@ function umoci() {
 function gomtree() {
 	local args=("$@")
 
-	# We're rootless. Note that this is _not_ available from the upstream
-	# version of go-mtree. It's a feature I implemented in the library for
-	# umoci's support, and is currently being proposed in
-	# https://github.com/vbatts/go-mtree/pull/96. I would not hold my breath
-	# that it'll be merged any time soon.
```

Comment:
This comment is inaccurate, but it is still true that `-rootless` is [an (open)SUSE extension to `gomtree`](https://build.opensuse.org/package/view_file/Virtualization:containers/go-mtree/0001-gomtree-add-rootless-flag.patch?expand=1). The linked PR has changed since I wrote the commend, and it just adds an `FsEval` interface to all of the key methods -- but the integration with `umoci`'s `pkg/unpriv` is still out-of-tree.

Reply:
oops, misread. Updated..